catalog/resource_sql_table.go

```diff
@@ -24,6 +24,7 @@ var MaxSqlExecWaitTimeout = 50
 type SqlColumnInfo struct {
 	Name     string `json:"name"`
 	Type     string `json:"type_text,omitempty" tf:"alias:type,computed"`
+	Identity string `json:"identity,omitempty" tf:"default:false"`
```

Comment:
We can use the IdentifyColumn here 
```
Identity IdentityColumn `json....`
```
and then there is no need to convert types below.
In that case, you will need to create a 3rd const below for the default case.

Reply:
Done